### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (#blocklySearchInputField)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2269]
#### Issue
`Form elements must have labels` ([label](https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](http://localhost:3232/index.html#editor)
#### Element path
#blocklySearchInputField
#### Snippet
`<input type="text" placeholder="Search..." id="blocklySearchInputField" class="blocklySearchInputField" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
